MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -4893,47 +4859,52 @@ private void gabbtv()
         /// </summary>
         private void strtok()
         {
-            var string2SplitPointer = GetParameterPointer(0);
+            var stringToSplitPointer = GetParameterPointer(0);
             var stringDelimitersPointer = GetParameterPointer(2);
 
-            if (!Module.Memory.TryGetVariablePointer("STROK-RESULT", out var resultPointer))
-                resultPointer = Module.Memory.AllocateVariable("STROK-RESULT", 0x400);
-
-            if (!Module.Memory.TryGetVariablePointer("STROK-ORIGINAL", out var originalPointer))
-                originalPointer = Module.Memory.AllocateVariable("STROK-ORIGINAL", 0x400);
-
-            if (!Module.Memory.TryGetVariablePointer("STROK-ORDINAL", out var ordinalPointer))
-                ordinalPointer = Module.Memory.AllocateVariable("STROK-ORDINAL", 0x2);
+            var workPointerPointer = Module.Memory.GetOrAllocateVariablePointer("STRTOK-WRK", 4);
```

Comment:
I started using the `IntPtr16.Size` const on declarations just to denote what'll be saved there